src/components/Shifting/ShiftingBlock.tsx

```
@@ -123,25 +123,28 @@ export default function ShiftingBlock(props: {
+                    ["DistrictAdmin", "StateAdmin"].includes(
+                      authUser.user_type,
+                    ) ||
+                    authUser.home_facility_object?.id ===
+                      shift.assigned_facility
+                  )
+                }
+                onClick={onTransfer}
+              >
+                {t("transfer_to_receiving_facility")}
+              </button>
+            </>
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.
Unsafe fix: Remove the Fragment


(lint/complexity/noUselessFragments)